library/agent/hooks/isBuiltinModule.test.ts

```diff
@@ -0,0 +1,12 @@
+import * as t from "tap";
+import { isBuiltinModule } from "./isBuiltinModule";
+
+t.test("it works", async (t) => {
+  t.equal(isBuiltinModule("fs"), true);
+  t.equal(isBuiltinModule("mysql"), false);
+  t.equal(isBuiltinModule("http"), true);
+  t.equal(isBuiltinModule("node:http"), true);
+  t.equal(isBuiltinModule("test"), false);
+  // @ts-expect-error Testing the undefined case
+  t.equal(isBuiltinModule(undefined), false);
```

Comment:
```suggestion
  t.equal(isBuiltinModule(""), false);
```

Maybe better to check empty string, we would never be able to pass undefined to this function (typescript)?